### DISCUSSION TITLE
Trivial suggestion regarding fun quotes

### DISCUSSION THREAD

**Discussion by stillwoods** [janoside/btc-rpc-explorer#483] [Ideas]
I thought arranging the fun quotes chronologically might be a bit more interesting.  Either from oldest at top to newest at bottom or the reverse.  If new quotes are added, they would be easier to find as well with chronological organization.

**Comment by janoside** [comment]
@stillwoods Thanks for the suggestion. I liked it and it's done in master (no official release yet, but you can see it in action [here](https://bitcoinexplorer.org/quotes)).
